File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/K. Max and Min/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/K

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemK
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' '); // take one line input

            int x = Convert.ToInt32(input[0]);
            int y = Convert.ToInt32(input[1]);
            int z = Convert.ToInt32(input[2]);
            int min = Math.Min(Math.Min(x, y), z);
            int max = Math.Max(Math.Max(x, y), z);
            Console.WriteLine($"{min} {max}");
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/K. Max and Min/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/K

#include<iostream>
using namespace std;
 
int main(){
    long long x, y, z, min, max;
    cin >> x >> y >> z;
    min = x > y? y : x;
    max = x > y? x : y;
    if(min > z) min = z;
    if(max < z) max = z;
 
    cout << min << ' ' << max;
}
